main.cpp

```diff
@@ -10,158 +10,224 @@ video: Chapter 2 - Part 3
-5) pass each of your function parameters to the ignoreUnused function like you did in b)
-6) if your function returns something other than void, add 'return { };' at the end of it.
-7) provide default values for an arbitrary number of parameters in the function parameter list.
-
-8) consult the coding style guide found in the Readme.MD and adjust the formatting of your 10 functions.  At this point, you might have something that looks like this:
-    float someFunc2(bool yes, int bar=2) { ignoreUnused(yes, bar ); return { }; }
-    This does not conform with the coding standard for this course (check the Readme.MD) and needs to be corrected
-    
-9) in the main function at the end: 
-    for each of those functions declared, 
-        a) write out how the function would look if called with correct arguments
-        b) if the function returned anything, store it in a local variable via the 'auto' keyword.
-        c) pass the local variables to ignoreUnused() as you did in 2b)
-    see main() for an example of this.
- 
+4) add { ignoreUnused( ); } after each declaration in place of the closing
+semicolon 5) pass each of your function parameters to the ignoreUnused function
+like you did in b) 6) if your function returns something other than void, add
+'return { };' at the end of it. 7) provide default values for an arbitrary
+number of parameters in the function parameter list.
+
+8) consult the coding style guide found in the Readme.MD and adjust the
+formatting of your 10 functions.  At this point, you might have something that
+looks like this: float someFunc2(bool yes, int bar=2) { ignoreUnused(yes, bar );
+return { }; } This does not conform with the coding standard for this course
+(check the Readme.MD) and needs to be corrected
+
+9) in the main function at the end:
+    for each of those functions declared,
+        a) write out how the function would look if called with correct
+arguments b) if the function returned anything, store it in a local variable via
+the 'auto' keyword. c) pass the local variables to ignoreUnused() as you did in
+2b) see main() for an example of this.
+
 10) click the [run] button.  Clear up any errors or warnings as best you can.
  */
 
-//2)
-void variableDeclarations()
-{
-    //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+// 2)
+
+void variableDeclarations() {
+  // example:
+  int number =
+      2; // declaration of a variable named "number", that uses the primitive
+         // type 'int', and the variable's initial value is '2'
+  float price = 20.f;
+  double coin = 0.3;
+  bool isTrue = true;
+  char alphabet = 'a';
+  unsigned int someNum = 2;
+
+  ignoreUnused(
+      number, price, coin, isTrue,
+      someNum); // passing each variable declared to the ignoreUnused() function

```

Comment:
```
2) for each primitive type, write out 3 variable declarations inside the
variableDeclaration() function on line 59. a) give each variable declaration an
initial value
```
I'm expecting to see 3 integers, 3 doubles, 3 bools, 3 floats, and 3 'char' types. 
I only see one of each. 

- [ ] revise